### PR TITLE
Adding the logTimeFormat value for time formatting in the KEDA Operator

### PR DESCRIPTION
**Pull request by tbickford**
This allows for adjusting the `zap-time-encoding` argument to the KEDA Operator so that time formats like ISO8601 can be used. ISO8601 is more reader-friendly when examining the timestamps in the logs and possibly could be used as the default value in the values.yaml.

### CODE REVIEW

**Comment by zroubalik** [comment]
@tbickford thanks, this is great. Could you please open another PR in kedacore/keda repo to add this change there as well?

**Comment by zroubalik** [comment]
We should fix this https://github.com/kedacore/charts/issues/42 before mergin this PR, to include you change to the latest v.1.4 release.

File: keda/Chart.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ description: Event-based autoscaler for workloads on Kubernetes
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 1.3.2
+version: 1.3.3
```
Comment (tomkerkhove) [inline keda/Chart.yaml:7]: This should be 1.4.0 now, please merge master.
```suggestion
version: 1.4.0
```

**Comment by tomkerkhove** [comment]
For the rest it looks good to me, thanks!

**Comment by tomkerkhove** [comment]
@tbickford We provide some help with DCO issues on https://github.com/kedacore/keda/blob/master/CONTRIBUTING.md#i-didnt-sign-my-commit-now-what

**Comment by tbickford** [comment]
@tomkerkhove Thanks for the link.

**Comment by tomkerkhove** [comment]
My pleasure, thank you!

**Pull request merged**